CHANGELOG.md

```diff
@@ -11,12 +11,14 @@ and this project adheres to [Semantic Versioning](https://semver.org/spec/v2.0.0
 ### Fixed
 
 - Don't panic when AWSControlPlane CR AZs are nil.
+- Add suffix to Route Tables to get rid of naming collision.
 
 ### Changed
 
 - Updated workflows for automatic release.
 - Updated backward incompatible Kubernetes dependencies to v1.18.5.
 - Removed migration code to ensure the Control Plane CRs for existing Node Pool clusters.
+- Add Route Table lookup using tags, so `RouteTables` flag can be phased out in the future.
```

Comment:
There is a `### Deprecated` section. We can add a note about `RouteTables` flag there.

Till we don't break the compatibility I think we can add this entry to `### Added` and once we remove `RouteTables` it will get added to `### Removed` and that will be the breaking change.

Headers.